src/test/java/com/zaxxer/nuprocess/EnvironmentTest.java

```diff
@@ -69,16 +68,13 @@ public void defaultEnv() throws InterruptedException, IOException
 		ProcessBuilder pb = new ProcessBuilder(command);
 		pb.environment().clear();
 		pb.environment().putAll(env);
-		System.out.println("Started Java Process");
-		Process process = pb.start();		
-		System.out.println("Waited for Java Process");
-		Set<String> result = new HashSet<String>();
```

Comment:
Since these results are going to be compared using `assertEquals`, hash order vagaries concern me. I haven't seen any failures, but a more consistent ordering doesn't seem like a bad idea. Happy to revert, if you disagree (or to switch to `TreeSet` if you think we shouldn't care about the actual order, and only about what variables are set)